config/settings/production.py

```diff
@@ -13,9 +13,10 @@
 DATABASES["default"] = env.db("DATABASE_URL")  # noqa F405
 DATABASES["default"]["ATOMIC_REQUESTS"] = True  # noqa F405
 DATABASES["default"]["CONN_MAX_AGE"] = env.int("CONN_MAX_AGE", default=60)  # noqa F405
-if env("POSTGRES_SSL"):
+if env.bool("POSTGRES_SSL"):
     DATABASES["default"]["OPTIONS"] = {"sslmode": "require"}  # noqa F405
-
+elif not env.bool("POSTGRES_SSL"):
```

Comment:
Not just else?

Reply:
Yeah, that probably makes more sense, I had originally thought about raising an exception but since it's a bool, no need.